addons/l10n_jo_edi/wizard/account_move_send.py

```diff
@@ -0,0 +1,56 @@
+from odoo import models, fields, api
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = "account.move.send"
+
+    l10n_jo_edi_is_visible = fields.Boolean(
+        compute="_compute_l10n_jo_edi_is_visible",
+        help="Jordan: technical field to determine if the option to submit a Jordanian electronic invoice is visible.",
+    )
+    l10n_jo_edi_is_enabled = fields.Boolean(
+        string="Send JoFotara e-invoice",
+        compute="_compute_l10n_jo_edi_is_enabled",
+        store=True,
+        readonly=False,
+        help="Jordan: used to determine whether to submit this e-invoice.",
+    )
+
+    @api.depends("move_ids")
+    def _compute_l10n_jo_edi_is_visible(self):
+        for wizard in self:
+            wizard.l10n_jo_edi_is_visible = any(move.l10n_jo_edi_is_needed and move.l10n_jo_edi_state == 'to_send' for move in wizard.move_ids)
```

Comment:
It's wrong by design. Since you can untick the `l10n_jo_edi_is_enabled` checkbox, your invoices must not be already `to_send`.